src/pages/iv-ava-by-site/index.js

```diff
@@ -0,0 +1,217 @@
+import { useContext } from 'react'
+import Table from 'src/components/Shared/Table'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { useWindow } from 'src/windows'
+import { useResourceQuery } from 'src/hooks/resource'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import RPBGridToolbar from 'src/components/Shared/RPBGridToolbar'
+import SerialForm from './forms/SerialForm'
+import serialIcon from '../../../public/images/TableIcons/imgSerials.png'
+import lotIcon from '../../../public/images/TableIcons/lot.png'
+import { Box, IconButton } from '@mui/material'
+import Image from 'next/image'
+import LotForm from './forms/LotForm'
+
+const AvailabilitiesBySite = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { stack } = useWindow()
+
+  async function fetchGridData(options = {}) {
+    const { _startAt = 0, _pageSize = 50, params } = options
+
+    const response = await getRequest({
+      extension: `RG.IV.asmx/IV403_p`,
```

Comment:
no hard code please, add 'ReportIVGenerator' repository and the api if they don't exist